resources/views/Admin/groups/show.blade.php

```diff
@@ -0,0 +1,62 @@
+@extends('layouts.layout', ['title' => 'Group Management'])
+
+@section('sidebar')
+    @include('layouts.sidebar', ['sidebar'=> Menu::get('sidebar_admin')])
+@endsection
+
+@section('content')
+<!doctype html>
+
+<div class="container mt-4">
+      <div class="card card-body">
+        <div class="container">
```

Comment:
Container is not needed

Reply:
If container is removed - design does not match mock up
<img width="1648" alt="screen shot 2018-10-03 at 11 04 01 am" src="https://user-images.githubusercontent.com/29666170/46429736-115a7800-c6fc-11e8-8b52-a5bb8f34c79c.png">

Reply:
<img width="1533" alt="screen shot 2018-10-03 at 11 04 30 am" src="https://user-images.githubusercontent.com/29666170/46429772-22a38480-c6fc-11e8-9ef2-780fd190d2d0.png">

with container